index.html

```diff
@@ -0,0 +1,93 @@
+                    <li class="padding"><a class="pad" href="./about.html">About</a></li>
+                    <li class="padding"><a class="pad" href="#">Contact</a></li>
+                    <li class="padding"><a class="pad" href="#">Join</a></li>
+                  </ul>
+
+                <!-- desktop menu -->
+                <div class="desktop-menu">
+                    <img id="logo" src="./img/logo.jpg" alt="logo of company">
+                    <a href="./pages/index.html">Home</a>
+                    <a href="./about.html">About</a>
+                    <a href="#">Program</a>
+                    <a href="#">Join</a>
+                    <a href="#">Sponsor</a>
+                    <a href="#">Campaign</a>
+
+                </div>
+                
+            </nav>
+        </div>
+        <header class="flex-column  bg">
+        
+            <div class="margin-header">
+                <h3 class="orange-color" id="header-text-1">
+                    "Hello people from all round the world"
+                </h3>
+                <h1 class="orange-color header-text2">GET INSPIRED BY THE BEST PEOPLE FROM AROUND THE WORLD ABOUT CLIMATE CHANGE</h1>
+            </div>
+            <p class="paragraph">This is Commitment to the people who have worked day and night to pass.</p>
+            <div class="margin-header ">
+                <h3 class="hashtags">#saveEarth #GreenCare #development</h3>
+                <h4 class="last-header-text ">@ For a better tomorrow that we will work for our kids </h4>
+            </div>
+            </header>
+
+        <section class="section-topic">
+            <h1 id="title-section-2">Topic</h1>
+  <div id="desktop-variation-1">
+      <article class="article margin-header"><img class="article-img" src="./img/emb.png" alt="icon"><h3 class="text-overflow article-text ">Eco system design</h3>
+      </article>
+      <article class="article margin-header"><img class="article-img" src="./img/Protocol.png" alt="icon"><h3 class="text-overflow article-text">Protocols for effective reach</h3>
+      </article>
+      <article class="article margin-header"><img class="article-img" src="./img/OS.png" alt="icon"><h3 class="text-overflow article-text"> Operating Systems for predictive data </h3>
+      </article>
+      <article class="article margin-header"><img class="article-img" src="./img/progra.png" alt="icon"><h3 class="text-overflow article-text">Programming approach</h3>
+      </article>
+      <article class="article margin-header"><img class="article-img" src="./img/algo.png" alt="icon"><h3 class="text-overflow article-text">Algorithms</h3>
+      </article>
+
+  </div>
+        
+            
+
+        </section>
+        <section class="speakers-section"> 
+            <h1 id="speaker-head">Speakers</h1>
+            <div class="bar"></div>
+            <div class="desktop-v"></div>
+      
+    </section>
```

Comment:
- [ ] Well done implementing the two columns. However, the content is still more to the left side, kindly make it more centered like in the image below 👇 
- Original design 👇 
![Screenshot (253)](https://user-images.githubusercontent.com/67087664/164892193-29d6530d-0235-42f4-a75a-770afb267d2c.png)